Update the_problem.md (#377)
cowprotocol/docs: docs/cow-protocol/reference/core/auctions/the_problem.md

where $$p(s)$$ is the price of the sell token relative to a numéraire and is externally provided. Also here, orders can be executed over multiple batches.

## Protocol Fees

Each user order may have an associated fee paid to the protocol. At a high level, these fees can be represented by a function that, for a given order $$S$$ maps all possible trades to a non-negative vector of tokens, that is $$f_S:S \rightarrow \mathbb R^k_+$$   with $$f_S(0)=0$$.

> Note that solvers are also expected to charge their fee to cover the cost of executing an order. We discuss such fees later in the context of solvers' optimal bidding, but we do not account for them here as they are not part of the protocol.

Reviewer: ```suggestion
:::note

Solvers are also expected to charge an additional fee to cover the cost of executing an order. We discuss such fees later in the context of solvers' optimal bidding, but not here as they are not part of the protocol.

:::
```
When doing notes, it is advised to use admonitions within Docusaurus

Author: I did a slight rewrite to your suggestion but apparently I cannot commit it

Reviewer 2: Pushed the suggestion by mfw
